open_academy/controllers/controllers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please don't add commented code. Remove this file and all the references to it. When you need this you'll add it.